remoteworkitem/trackerquery_repository.go

```diff
@@ -48,11 +53,25 @@ func (r *GormTrackerQueryRepository) Create(ctx context.Context, query string, s
 		return nil, InternalError{simpleError{err.Error()}}
 	}
 
+	spaceType := "spaces"
+	spaceSelfURL := rest.AbsoluteURL(goa.ContextRequest(ctx), app.SpaceHref(spaceID.String()))
 	tq2 := app.TrackerQuery{
 		ID:        strconv.FormatUint(tq.ID, 10),
 		Query:     query,
 		Schedule:  schedule,
-		TrackerID: tracker}
+		TrackerID: tracker,
+		Relationships: &app.TrackerQueryRelationships{
```

Comment:
It looks like the following code block is repeated in each repository method and I wonder if you can outsource it into a function.